packages/fast-component-explorer/app/explorer.tsx

```diff
@@ -0,0 +1,388 @@
+                                            label={"documentation"}
+                                            items={this.renderPivotItems()}
+                                            jssStyleSheet={{
+                                                pivot: {
+                                                    height: "100%",
+                                                },
+                                                pivot_tabPanels: {
+                                                    height: "100%",
+                                                    overflow: "auto",
+                                                    padding: "0 12px",
+                                                },
+                                            }}
+                                        />
+                                    </Background>
+                                </Row>
+                            </Canvas>
+                            <Pane resizable={true} resizeFrom={PaneResizeDirection.west}>
+                                <div
+                                    style={{
+                                        overflow: "auto",
+                                        minHeight: "100%",
+                                        background: "#212121",
+                                    }}
+                                >
+                                    <Form
+                                        data={this.state.scenario.props}
+                                        onChange={this.handleUpdateData}
+                                        schema={this.resolveSchemaById(
+                                            get(this.state.scenario, "id")
+                                        )}
+                                        location={{
+                                            onChange: this.handleUpdateLocation,
+                                            dataLocation: this.state.dataLocation,
+                                        }}
+                                        childOptions={childOptions}
+                                        jssStyleSheet={this.formStyleOverrides}
+                                    />
+                                </div>
+                            </Pane>
+                        </Row>
+                    </Container>
+                </Background>
+            </DesignSystemProvider>
+        );
+    }
+
+    private renderDirectionToggle(): React.ReactNode {
+        return (
+            <button onClick={this.handleUpdateDirection}>
+                {this.state.viewConfig.direction}
+            </button>
+        );
+    }
+
+    private renderPivotItems(): TabsItem[] {
+        return [
+            {
+                tab: (className: string): React.ReactNode => "Code Preview",
+                content: (className: string): React.ReactNode => {
+                    return (
+                        <div className={className}>
```

Comment:
Looks like you can't fully scroll the contents of the syntax highlighter when the row height is smaller:
![image](https://user-images.githubusercontent.com/37851214/60843740-59776100-a18c-11e9-8571-290511a444ec.png)

just add `height: calc(100% - 32px)` to you `pivot_tabPanels` class